doc/tutorials/debug.rst

```diff
@@ -214,16 +204,9 @@ shown in the following example:
       inserted trace information
 
 4. After we have inserted the trace code addition, we need to rebuild
-   the acrn-hypervisor in the ``acrn-hypervisor`` directory::
-
-      # make PLATFORM=uefi
-
-   and replace the existing ``acrn.efi`` with the new one we just
-   built in the ``build/hypervisor/`` folder::
-
-      # mount /dev/sda1 /mnt && mkdir -p /mnt/EFI/acrn
-      # mv /mnt/EFI/acrn/acrn.efi /mnt/EFI/acrn/acrn.efi.old
-      # cp build/hypervisor/acrn.efi /mnt/EFI/acrn/
+   the ACRN hypervisor and install it on the platform. Refer to
```

Comment:
I'm not that fond of sending folks off to a long document (and in this case two documents) to find the commands to use, but I'll go with the suggestion.

Reply:
I don't disagree with you, I hesitated a bit before doing this. What decided me was the fact that the instructions in this doc are already incorrect (`make PLATFORM=uefi` has been removed not long ago)... and I'm sure this will happen again.